### PR TITLE
ts: Migrate service to Horizon project

### LINKED ISSUE #6480: Ptm.Ts.IMeasurementServer: 4 (OpenSession) is not implemented
### Service call

Ptm.Ts.IMeasurementServer: 4 (OpenSession) is not implemented.

[Switchbrew](https://switchbrew.org/wiki/PTM_services#OpenSession_3) | [hbmenu code](https://github.com/switchbrew/nx-hbmenu/pull/162/files)

### Service description

```
Takes an input DeviceCode. Returns an #ISession.
```

### Required by

https://github.com/Ryujinx/Ryujinx-Games-List/issues/3925 (v3.6.0)

### PR DESCRIPTION
**Pull request by AcK77**
This PR migrate the `ts` service (stored in `ptm`) to the Horizon project:
- It stubs all known IPCs.
- IpcServer consts are checked by RE.

Closes #6480

### CODE REVIEW

File: src/Ryujinx.Horizon/Ptm/Ipc/MeasurementServer.cs [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        {
+            Logger.Stub?.PrintStub(LogClass.ServicePtm, new { location });
+
+            temperatureMilliC = DefaultTemperature * 1000;
+
+            return Result.Success;
+        }
+
+        [CmifCommand(4)] // 8.0.0+
+        Result IMeasurementServer.OpenSession(DeviceCode deviceCode, out ISession session)
```
Comment (gdkchan) [inline src/Ryujinx.Horizon/Ptm/Ipc/MeasurementServer.cs:56]:
```suggestion
        public Result IMeasurementServer.OpenSession(DeviceCode deviceCode, out ISession session)
```

**Review by gdkchan — COMMENTED** [review]
I assume we will need to change `TsMain` to `PtmMain` in the future?

**Comment by AcK77** [comment]
> I assume we will need to change `TsMain` to `PtmMain` in the future?

Yeah it's the same as Audio service.

**Review by gdkchan — APPROVED** [review]
lgtm, thanks.

**Comment by TSRBerry** [comment]
> > I assume we will need to change `TsMain` to `PtmMain` in the future?
> 
> Yeah it's the same as Audio service.

Could we add a note/todo comment for these?

**Review by TSRBerry — APPROVED** [review]
Lgtm! Could add the comments for the classes that we need to rename at some point before you merge this?

**Comment by gdkchan** [comment]
The code seems to indicate it is supposed to be 42ºC, not 0, so it seems there is something wrong with the code.

**Comment by TSRBerry** [comment]
works fine here, although I'm using nx-hbloader v2.4.4

**Pull request merged**